tests/integration/framework/process/daprd/daprd.go

```diff
@@ -120,6 +130,9 @@ func New(t *testing.T, fopts ...Option) *Daprd {
 	if opts.appHealthCheckPath != "" {
 		args = append(args, "--app-health-check-path="+opts.appHealthCheckPath)
 	}
+	if len(opts.componentFiles) > 0 {
```

Comment:
Nit: Consider renaming to `resourceFiles` to support all kind of resources

Reply:
Done